packages/eslint-config-vivy/index.js

```diff
@@ -1,19 +1,24 @@
 module.exports = {
-  parser: 'babel-eslint',
+  parser: '@typescript-eslint/parser',
   extends: [
     'airbnb',
     'prettier',
     'prettier/react',
-    "plugin:jest/recommended"
-],
-  "plugins": [
-    "prettier",
-    "jest"
+    'plugin:react/recommended',
+    'plugin:jest/recommended',
   ],
+  plugins: ['prettier', 'jest', 'react'],
   rules: {
-    "class-methods-use-this": 0,
-    "import/prefer-default-export": 0,
-    "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
-    "prettier/prettier": ["error"]
-  }
+    'class-methods-use-this': 0,
+    'import/prefer-default-export': 0,
+    'react/jsx-filename-extension': [1, { extensions: ['.js', '.ts', '.jsx'] }],
+    'prettier/prettier': ['error'],
+  },
+  parserOptions: {
+    ecmaVersion: 6,
```

Comment:
but what if we have to use ECMA2018 for example ? 🤔

Reply:
good question, @Weetbix what do you think?

Reply:
I guess we can set this to the highest available, as there are never breaking changes backwards right? and use the year naming scheme,  so `2020`?

Reply:
I think we can use it, but I think also it will override the `ecmaversions` from the packages which we extended here 😉

Reply:
but I think we can get rid of it as we are extending `airbnb` plugin, no ?

Reply:
Im happy to keep it as 6 until we run into an issue with it. We're writing typescript so I imagine this will be irrelevant most of the time anyway.